.github/workflows/ci.yml

```
@@ -40,6 +47,58 @@ jobs:
+          mode: compare
+          branch: ${{ github.event.pull_request.base.ref }}
+
+      - name: Store Bundle Stats
+        if: github.event.pull_request.base.ref == 'next'
+        uses: ./.github/workflows/bundle-stats
+        with:
+          mode: store
+          branch: ${{ github.event.pull_request.base.ref }}
+
+      - name: Update PR Description
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)